### DISCUSSION TITLE
Copilot takes my code too literally

### DISCUSSION THREAD

**Discussion by gummz** [community/community#31168] [Copilot Conversations]
I often include just a "pass" statement in my empty Python functions because red squiggly lines annoy me. Now, after a few of those, all Copilot suggests now for my functions is "pass".

**Comment by MulleDK19** [comment]
Well, you're conditioning it to put 'pass' in functions. If it didn't, you couldn't use it for specific tasks.

**Reply by gummz** [reply]
Indeed, but now all the suggestions are "pass". It would be fine if only one of them was.

**Comment by DenverCoder1** [comment]
You can see if it helps to check for alternate suggestions.

<kbd>Alt</kbd>+<kbd>[</kbd> and <kbd>Alt</kbd>+<kbd>]</kbd> can be used to cycle through suggestions, or <kbd>Ctrl</kbd>+<kbd>Enter</kbd> to open a pop-up tab with suggestions.

https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code
